lib/api/util.js

```diff
@@ -26,13 +28,13 @@ async function getResolveErrorBodyCallback ({ callback, body, contentType, statu
 
   try {
     if (contentType.startsWith('application/json')) {
-      const payload = JSON.parse(toUSVString(Buffer.concat(chunks)))
+      const payload = JSON.parse(chunksDecode(chunks, length))
```

Comment:
I think this is false. If I understand it correctly chunksDecode second parameter is about the amount of array items and not the total length of the whole Buffer.

Reply:
No. The length of the buffer.

https://nodejs.org/docs/latest/api/buffer.html#static-method-bufferconcatlist-totallength